todo-app/src/todoState.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+
+class todoState extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            checked: this.props.done
+        }
+        this.handleChange = this.handleChange.bind(this);
+    }
+
+    handleChange(event) {
+        let changed = !this.state.checked
+        this.setState({
```

Comment:
Great that you used `setState` here. But you should either use `event.target.value` to be sure you are using the most updated checked value. Or use the previous state value, for example:

```js
this.setState(prevState => ({
  checked: !prevState.checked
}));
```

to avoid any issues with [asynchronous state updates](https://reactjs.org/docs/state-and-lifecycle.html#state-updates-may-be-asynchronous).